pawar-2018/pawar2018
# Query opts and cachebusting JS

# What does this pull request do?
- Cachebusts the JS file
- Moves the resets for events after the While loop ends

# How should the reviewer test this pull request?
- Pull it down and see if the JS is still loaded but with a random string appended to it.
- Events still works fine, maybe even faster.

# Include the Trello card for this PR, if there is one.
N/A

# Include any screenshots that will help explain this PR.
N/A

# Any another context you want to provide?
🐙
